[fix #8] [mozmeao/infra#1104] Clean up oregon-a stage yaml.

jpetto opened the pull request.

jpetto committed:
[fix #8] [mozmeao/infra#1104] Clean up oregon-a stage yaml.

jgmize commented on oregon-a/basket-stage/web-deploy.yaml:
```diff
@@ -48,286 +37,190 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-stage-v100-env
+          value: 3
```
```suggestion
          value: "3"
```

jgmize commented on oregon-a/basket-stage/web-deploy.yaml:
```diff
@@ -48,286 +37,190 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-stage-v100-env
+          value: 2
```
```suggestion
          value: "2"
```

jgmize commented on oregon-a/basket-stage/web-deploy.yaml:
```diff
@@ -48,286 +37,190 @@ spec:
-              key: wsgi-num-workers
-              name: basket-stage-v100-env
-        image: 127.0.0.1:5555/basket-stage:v100
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-a/basket-stage/web-deploy.yaml:
```diff
@@ -48,286 +37,190 @@ spec:
-              name: basket-stage-v100-env
-        image: 127.0.0.1:5555/basket-stage:v100
+          value: 4
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize commented on oregon-a/basket-stage/worker-deploy.yaml:
```diff
@@ -48,286 +37,190 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-stage-v100-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-a/basket-stage/worker-deploy.yaml:
```diff
@@ -48,286 +37,190 @@ spec:
-              key: wsgi-num-workers
-              name: basket-stage-v100-env
-        image: 127.0.0.1:5555/basket-stage:v100
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-a/basket-stage/worker-deploy.yaml:
```diff
@@ -48,286 +37,190 @@ spec:
-              name: basket-stage-v100-env
-        image: 127.0.0.1:5555/basket-stage:v100
+          value: 4
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jpetto committed:
Review fixes:

- remove superfluous data from ns.yaml
- quote all numeric env var values
- use full image hash
